src/shop-app.html

```diff
@@ -18,11 +18,25 @@
 <link rel="import" href="../bower_components/iron-media-query/iron-media-query.html">
 <link rel="import" href="../bower_components/iron-pages/iron-pages.html">
 <link rel="import" href="../bower_components/iron-selector/iron-selector.html">
+<link rel="import" href="../bower_components/paper-icon-button/paper-icon-button.html">
```

Comment:
paper-icon-button is lazily imported by `lazy-resources.html` and should not be imported here.

This is also the case for the other added imports in this PR.